raiwidgets/raiwidgets/responsibleai_dashboard_input.py

```diff
@@ -171,7 +171,7 @@ def _prepare_filtered_error_analysis_data(self, features, filters,
 
     def debug_ml(self, data):
         try:
-            features = data[0]
+            features = data[0]  # TODO: Remove prompt feature
```

Comment:
I think comments need to be in-line to pass linting - but in this case I think we can even just remove this comment?

Reply:
There are more changes to be made in this method that are not a part of this PR. This is just a marker for making sure those are accounted for in the next PR. Will remove it then. Looks like this comment did pass linting so don't need to worry about that.